Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/openjdk:8-42
+FROM registry.opensource.zalan.do/stups/openjdk:8-cd26
```

Comment:
One question that should probably have been asked on the stups training in the morning: could we use the latest tag here or do we always have to find the latest tag and then pinpoint it?